docs/content/guides/dagster/code-references.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Linking to assets' code with code references | Dagster Docs
+---
+
+# Linking to assets' code with code references
+
+<Note>
+  This feature is considered <strong>experimental</strong> and is under active
+  development. This guide will be updated as we roll out new features.
+</Note>
+
+Attaching code reference metadata to your Dagster assets allows you to easily view those assets' source code from the Dagster UI:
+
+- **In local development**, navigate directly to the code backing an asset in your editor
+- **In Dagster+**, link to source code in your source control repository to see the full change history for an asset
+
+In this guide, we'll show you how to automatically and manually attach code references to your Dagster assets.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- A Dagster repository with assets that you want to link to code
```

Comment:
"repository" is mostly a deprecated term. Maybe just "a set of Dagster asset definitions"?